tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -258,6 +259,7 @@ def test_nodereplacement_proactive_with_io_running(
 @skipif_bmpsi
 @skipif_external_mode
 @skipif_ms_consumer
+@skipif_hci_client
```

Comment:
Maybe we can just check if it's HCI muliticluster execution, if so, to enforce switching the context of config for provider, run tests on provider clutster and return back context of config back to origin after the test will finis.   But for now I am ok to disable those, but in future we can cover those cases like this I think.